README.md

```diff
@@ -13,9 +25,9 @@ $ reliquery-ui
 ```
 Navigate to http://localhost:8000/reliquery/demo/tutorial/intro to view Relics
 
-## For development
+## For development<a name="dev"></a>
 
-### Local Install
+### Local Install<a name="loc"></a>
```

Comment:
This is for like README improvement? I dont know what this does off the top fo my head, but if it works thats awesome!

Reply:
Yeah this connects the table of contents to the this content